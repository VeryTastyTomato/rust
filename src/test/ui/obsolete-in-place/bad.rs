// Check that `<-` and `in` syntax gets a hard error.

fn main() {
    let (x, y, foo, bar) = (0, 0, 0, 0);
    x <- y; // ok: parses as a comparison
    in(foo) { bar }; //~ERROR expected expression, found keyword `in`
}
