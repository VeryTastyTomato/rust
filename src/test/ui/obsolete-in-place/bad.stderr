error: expected expression, found keyword `in`
  --> $DIR/bad.rs:6:5
   |
LL |     in(foo) { bar };
   |     ^^ expected expression

error: aborting due to previous error

